From: Ole Streicher <olebole@debian.org>
Date: Fri, 2 Dec 2016 11:11:28 +0100
Subject: Disable known test failures

Disable tLSQaips and tLSQFit tests

These tests fail, probably due to the test itself. From
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=838424#15

I'm not sure what to do about the other two tests that fail on arm64,
"tLSQaips" and "tLSQFit", because those two generate a load of output
text containing floating-point numbers and compare that text with an
expected output. It might not be so easy to make that work on all
architectures. Perhaps one could simply disable those two tests.
Preferably one would still run them, so that we can see what happens
in the logs, but not have the build fail if only those tests fail.
---
 casa/IO/test/CMakeLists.txt              | 4 ++--
 casa/OS/test/CMakeLists.txt              | 9 ++++-----
 casa/System/test/CMakeLists.txt          | 2 +-
 images/Images/test/CMakeLists.txt        | 2 +-
 lattices/LatticeMath/test/CMakeLists.txt | 2 +-
 scimath/Fitting/test/CMakeLists.txt      | 4 ++--
 tables/TaQL/test/CMakeLists.txt          | 4 ++--
 7 files changed, 13 insertions(+), 14 deletions(-)

diff --git a/casa/IO/test/CMakeLists.txt b/casa/IO/test/CMakeLists.txt
index 188a6c4..3187b60 100644
--- a/casa/IO/test/CMakeLists.txt
+++ b/casa/IO/test/CMakeLists.txt
@@ -3,9 +3,9 @@ tAipsIOCarray
 tAipsIO
 tBucketBuffered
 tBucketCache
-tBucketFile
+# tBucketFile # https://github.com/casacore/casacore/issues/73
 tBucketMapped
-tByteIO
+# tByteIO # https://github.com/casacore/casacore/issues/73
 tByteSink
 tByteSinkSource
 tFilebufIO
diff --git a/casa/OS/test/CMakeLists.txt b/casa/OS/test/CMakeLists.txt
index 520ceed..e5c9ee3 100644
--- a/casa/OS/test/CMakeLists.txt
+++ b/casa/OS/test/CMakeLists.txt
@@ -3,10 +3,10 @@ tCanonicalConversion
 tConversion
 tConversionPerf
 tDataConversion
-tDirectory
+# tDirectory # https://github.com/casacore/casacore/issues/73
 tDirectoryIterator
 tEnvVar
-tFile
+# tFile # https://github.com/casacore/casacore/issues/73
 tHostInfo
 tIBMConversion
 tLECanonicalConversion
@@ -14,10 +14,9 @@ tMemory
 tMemoryTrace
 tModcompConversion
 tMutex
-tPath
 tPrecTimer
-tRegularFile
-tSymLink
+# tRegularFile # https://github.com/casacore/casacore/issues/73
+# tSymLink # https://github.com/casacore/casacore/issues/73
 tTime
 tTimer
 tVAXConversion
diff --git a/casa/System/test/CMakeLists.txt b/casa/System/test/CMakeLists.txt
index 790da89..8c3c3ea 100644
--- a/casa/System/test/CMakeLists.txt
+++ b/casa/System/test/CMakeLists.txt
@@ -5,7 +5,7 @@ tAppInfo
 tAppState
 tCasarc01
 tChoice
-tObjectID
+# tObjectID # https://github.com/casacore/casacore/issues/986
 )

 foreach (test ${tests})
diff --git a/images/Images/test/CMakeLists.txt b/images/Images/test/CMakeLists.txt
index 4115c29..6865919 100644
--- a/images/Images/test/CMakeLists.txt
+++ b/images/Images/test/CMakeLists.txt
@@ -69,7 +69,7 @@ tMIRIADImage
 # tPagedImage2 # requires geodetic/Observatories
 # tRebinImage # requires geodetic/Observatories
 tSubImage
-tTempImage
+# tTempImage # https://github.com/casacore/casacore/issues/543
 )
 
 foreach (test ${tests})
diff --git a/lattices/LatticeMath/test/CMakeLists.txt b/lattices/LatticeMath/test/CMakeLists.txt
index 47e5c76..23f254f 100644
--- a/lattices/LatticeMath/test/CMakeLists.txt
+++ b/lattices/LatticeMath/test/CMakeLists.txt
@@ -11,7 +11,7 @@ tLatticeHistograms
 tLatticeMathUtil
 tLatticeSlice1D
 tLatticeStatistics
-tLatticeStatsDataProvider
+# tLatticeStatsDataProvider # tries to allocate a 1024*1024*1024 array, which is at least 4GiB 
 tLatticeTwoPtCorr
 # tLattStatsSpecialize
 )
diff --git a/scimath/Fitting/test/CMakeLists.txt b/scimath/Fitting/test/CMakeLists.txt
index 35d258f..de69738 100644
--- a/scimath/Fitting/test/CMakeLists.txt
+++ b/scimath/Fitting/test/CMakeLists.txt
@@ -3,8 +3,8 @@ dConstraints
 dLSQFit
 tFitGaussian
 tLinearFitSVD
-tLSQaips
-tLSQFit
+#tLSQaips
+#tLSQFit
 tNonLinearFitLM
 )
 
diff --git a/tables/TaQL/test/CMakeLists.txt b/tables/TaQL/test/CMakeLists.txt
index 4dc7140..840f8f9 100644
--- a/tables/TaQL/test/CMakeLists.txt
+++ b/tables/TaQL/test/CMakeLists.txt
@@ -36,8 +36,8 @@ tTaQLNode
 # Only test scripts, no test programs.
 set (testscripts
 ttaql
-tTableGramCretab
-tTableGramAlttab
+# tTableGramCretab # https://github.com/casacore/casacore/issues/541
+# tTableGramAlttab # https://github.com/casacore/casacore/issues/541
 tTableGramUpdate
 tTableGramMasked
 tTableGramNull
